README.md

```diff
@@ -8,13 +8,16 @@ The app lets you:
 - Create visualizations and maps in DHIS2 by combining the catchment areas with DHIS2 reporting data
 - Generate target population estimates for each catchment area to quickly calculate coverage estimates
 - See travel time to the health facilities within the catchment area to support outreach and session planning
-- Requirements: An account with Crosscut (a third party application) 
+- Requirements: An account with Crosscut (a third party application) and internet access
+
+Submit requests and issues here: [GitHub Issues](https://github.com/crosscutio/dhis2-crosscut-app/issues)
```

Comment:
I think we should include a link to a new issue: https://github.com/crosscutio/dhis2-crosscut-app/issues/new (it might make it easier for new users)